src/components/molecules/content/portable-text/CodeSwitcher.svelte

```
@@ -0,0 +1,62 @@
+			<button
+				class={`rounded-md px-3 py-1 text-sm font-medium transition 
+					${
+						activeLanguage === snippet.language
+							? 'bg-gray-800 text-white'
+							: 'text-gray-400 hover:bg-gray-700 hover:text-white'
+					}`}
+				onclick={() => setLang(snippet.language)}
+			>
+				{snippet.language}
+			</button>
+		{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)